tsconfig.base.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
This needs to have the paths and baseUrl setup to work with packages:

`https://github.com/ricklove/minter-sdk/blob/rick/generate-api-types/tsconfig.json`

Also, it would be good to clean this up and remove all the comments.

Reply:
vscode can't find the package imports without the tsconfig paths:

![image](https://user-images.githubusercontent.com/703880/112203928-1029a800-8be1-11eb-911f-ffdc1258d62d.png)

Reply:
Also, I've found it helpful to keep the paths in a separate file:

https://github.com/ricklove/rick-love-master/blob/master/tsconfig-paths.json

And import it into the base like this:

```
{
    "extends": "./tsconfig-paths",
    "compilerOptions": {
    ...
}
```

Reply:
Strange, mine doesn't have this issue. 
![image](https://user-images.githubusercontent.com/845717/112205253-d9986100-8bd1-11eb-9f1b-ae1909d49010.png)

I'll investigate.

Reply:
@ricklove — does this issue go away if you build the typescript in the `packages/minter-contracts` package?

```bash
cd packges/minter-contracts
yarn build
```

Reply:
Nevermind, this is working after I rerun `yarn`. I've been doing this in my projects for so long I didn't realize it wasn't needed.